go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/hashicorp/errwrap v1.1.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-secure-stdlib/strutil v0.1.1
-	github.com/hashicorp/vault/sdk v0.4.2-0.20220426194706-f8e907e0deda
+	github.com/hashicorp/vault/sdk v0.4.2-0.20220511000023-fa93782110bf
```

Comment:
This is currently pointing at my branch in Vault which has the SDK changes needed here. Note to self to update this once the SDK changes are merged.